screens/map/MapScreen.js

```diff
@@ -79,7 +86,13 @@ export default class MapScreen extends React.Component {
         longitude: location.coords.longitude,
         ...deltas,
       };
-      this.setState({ locationErrorMsg: null, location, region });
+      // Don't re-animate if we're using the default store
```

Comment:
I tried to make this work with the default store, but the resetting of the location in `orderStoresByDistance` for only the default store produced some really odd behavior, so I think it's ok to lose the animation for the default store.